py/desitarget/brightstar.py

```diff
@@ -0,0 +1,458 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+"""
+=====================
+desitarget.brightstar
+=====================
```

Comment:
Did you forget to add and commit a unit test for `desitarget.brightstar`?  You mention unit tests in the PR but the current unit tests never even import this file.